website/server/models/group.js

```diff
@@ -577,6 +578,18 @@ schema.statics.cleanGroupQuest = function cleanGroupQuest () {
   };
 };
 
+async function _updateUserWithRetries (userId, updates, numTry = 1) {
+  return User.update({_id: userId}, updates).exec((err, raw) => {
```

Comment:
http://stackoverflow.com/questions/30672265/why-can-i-not-chain-catch-when-calling-mongoose-model-create-in-node